pkg/distro/rhel8/distro.go

```diff
@@ -267,7 +267,7 @@ func newDistro(name string, minor int) *distribution {
 				"iwl6050-firmware",
 			},
 		},
-		BIOS:       true,
+		BIOS:       false,
```

Comment:
The `bareMetalX86Platform` isn't used only for edge.  It's also used for the image installer (see L279 below), which is an anaconda ISO that deploys a prebuilt image.  For this, we want to keep BIOS enabled, so we'll have to use a separate platform object for that.